### PR TITLE
Support Select 1

### PR DESCRIPTION
**Pull request by penghuo**
*Issue #, if available:* #64

*Description of changes:*
Support numeric literals.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java [hunk 1]
```diff
@@ -85,6 +85,10 @@ public static Field makeField(SQLExpr expr, String alias, String tableAlias) thr
             methodParameters.add(new KVValue(alias));
             methodParameters.add(new KVValue(scriptCode));
             return new MethodField("script", methodParameters, null, alias);
+        } else if (expr instanceof SQLIntegerExpr) {
```
Comment (arsen-es) [inline src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java]: is it only for integers? how about `SELECT 1.0 FROM myData`?

File: src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java [hunk 2]
```diff
@@ -191,4 +192,14 @@ public void split_field() throws Exception {
...
+    @Test
+    public void functionAssign() throws Exception {
+        String query = "SELECT 10 as key "+
+                "from " + TEST_INDEX_ACCOUNT + "/account limit 1";
+
+        assertThat(
+                executeQuery(query),
+                hitAny(kvInt("/fields/key/0", equalTo(10))));
+    }
 }
```
Comment (arsen-es) [inline src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java]: please add another test without assigning an alias to the field.

File: src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java (hunk 1 above)
Comment (penghuo) [reply src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java]: Good point! Add the support for numeric literals.

File: src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java (hunk 2 above)
Comment (penghuo) [reply src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java]: Done.

File: src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/parser/FieldMakerTest.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+import static org.junit.Assert.assertTrue;
+
+public class FieldMakerTest {
+
+    private static final String ALIAS = "a";
+
+    private static final String TABLE_ALIAS = "t";
+
+    @Test
+    public void makeFiled_assign() throws SqlParseException {
```
Comment (qreshi) [inline src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/parser/FieldMakerTest.java]: Typo: I believe you meant ```makeField_assign()```, same for the test below

Comment (abbashus) [reply src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/parser/FieldMakerTest.java]: Just to add, we should be consistent with Java naming convention. It should be `makeFieldAssign()` here.

Comment (penghuo) [reply src/test/java/com/amazon/opendistroforelasticsearch/sql/unittest/parser/FieldMakerTest.java]: Done.

**Pull request merged**